k8s/core/services.go

```diff
@@ -63,6 +67,31 @@ func (c *Client) GetService(svcName string, svcNS string) (*corev1.Service, erro
 	return c.core.Services(svcNS).Get(svcName, metav1.GetOptions{})
 }
 
+// ListServices list services using filters or list all if options are empty
+func (c *Client) ListServices(svcNamespace string, listOptions metav1.ListOptions) (*corev1.ServiceList, error) {
+	if err := c.initClient(); err != nil {
+		return nil, err
+	}
+
+	return c.core.Services(svcNamespace).List(listOptions)
+}
+
+// GetExternalIP gets the externalIP of a loadBalancer present in a given namespace, returns empty if LoadBalancer not
+// found
+func (c *Client) GetExternalIP(namespace string) (string, error) {
+	if err := c.initClient(); err != nil {
+		return "", err
+	}
+	svcs, err := c.ListServices(namespace, metav1.ListOptions{FieldSelector: "type=LoadBalancer"})
+	if err != nil {
+		return "", err
+	}
+	for _, svc := range svcs.Items {
+		return svc.Status.LoadBalancer.Ingress[0].IP, nil
```

Comment:
So the first service out of the list of services will always have correct IP? And why are we always taking slice 0 of Ingress? Should we have some more asserts here? I'm not too familiar with, but is this enough to assume that first service with Ingress[0] will always have what we need?